source/model.py

```diff
@@ -2447,7 +2447,7 @@ class EnumerationDeclarationBegin(SourceChunk):
 
     def __init__(self, enum, indent = ""):
         super(EnumerationDeclarationBegin, self).__init__(enum,
-            "Beginning of enumeration %s declaration" % enum.enum_name,
+            "Beginning of enumeration %s declaration" % enum,
```

Comment:
This changes logic and commit messages does not describe the reason.
In general, `enum_name != str(enum) == enum.name`.